class.jetpack.php

```diff
@@ -614,10 +614,11 @@ private function __construct() {
 		add_filter( 'jetpack_just_in_time_msgs', '__return_true', 9 );
 		add_filter( 'jetpack_just_in_time_msg_cache', '__return_true', 9);
 
-		// If enabled, point edit post and page links to Calypso instead of WP-Admin.
+		// If enabled, point edit post, page, and comment links to Calypso instead of WP-Admin.
 		// We should make sure to only do this for front end links.
-		if ( Jetpack_Options::get_option( 'edit_links_calypso_redirect' ) && ! is_admin() ) {
-			add_filter( 'get_edit_post_link', array( $this, 'point_edit_links_to_calypso' ), 1, 2 );
+		if ( Jetpack::get_option( 'edit_links_calypso_redirect' ) && ! is_admin() ) {
+			add_filter( 'get_edit_post_link', array( $this, 'point_edit_post_links_to_calypso' ), 1, 2 );
+			add_filter( 'edit_comment_link', array( $this, 'point_edit_comment_links_to_calypso' ), 1, 3 );
```

Comment:
I think it would be more appropriate to use [`get_edit_comment_link`](https://developer.wordpress.org/reference/hooks/get_edit_comment_link/) filter here instead, the same way as we do for edit post links.

Reply:
I... could have sworn I looked for that, and didn't find it 🙃 Yep, for sure, much easier.